controllers/workspace/eventhandlers.go

```diff
@@ -47,7 +47,13 @@ func dwRelatedPodsHandler(obj client.Object) []reconcile.Request {
 }
 
 func (r *DevWorkspaceReconciler) dwPVCHandler(obj client.Object) []reconcile.Request {
-	// Check if PVC is owned by a DevWorkspace (per-workspace storage case)
+	if obj.GetDeletionTimestamp() == nil {
+		return []reconcile.Request{}
+	}
+	// we can wrap the code below for handling per-workspace PVCs
```

Comment:
Thanks for reporting https://github.com/devfile/devworkspace-operator/issues/1250 -- I think this comment could be now simplified to something along the lines of:

```
// TODO: Ensure all new and existing PVC's get the `controller.devfile.io/devworkspace_pvc_type` label.
// See: https://github.com/devfile/devworkspace-operator/issues/1250
```